source/server.js

```diff
@@ -0,0 +1,33 @@
+import DocumentTitle from 'react-document-title'
```

Comment:
It would be nice to swap this out for helmet. But it's not a blocker for this. As there is a lot of other meta-data we'd usually want to include.